db/migrate/20220805041347_create_payments.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePayments < ActiveRecord::Migration[6.1]
```

Comment:
I hate the name `Payments`. This is the token of the card of the user so you can reuse it later. It isn't payment. Payment imply record of paying.